File: 1. Beginner/bee-1151.go
package main

import "fmt"

func main() {
	var count, sum, x, z int
	fmt.Scan(&x)

	for true {
		fmt.Scan(&z)
		if z > x {
			break
		}
	}

	for i := x; sum < z; i++ {
		sum += i
		count++
	}
	fmt.Println(count)
}

File: 1. Beginner/Java/bee-1151.java
import java.util.Scanner;

public class Main {
    public static void main(String[]args){
        Scanner sc = new Scanner (System.in);
        
        int x = sc.nextInt();
        int z;
        do{
            z = sc.nextInt();
        }while(z<=x);
        int sum = 0; int count = 0;
        for(int i = x; sum< z; i++){
            sum +=i;
            count++;
        }
        System.out.println(count);
    }
}